src/components/cylc/Toolbar.vue

```diff
@@ -197,9 +197,3 @@ export default {
   }
 }
 </script>
-
-<style>
-#core-app-bar a {
-  text-decoration: none;
-}
```

Comment:
Tested, and made no difference. I was googling for an image for the Dashboard for the icons, and found this old screenshot of Cylc UI :grimacing: 

![D8Vr_CkUEAAtxdr](https://user-images.githubusercontent.com/304786/92178870-ac839b80-ee97-11ea-9de6-feeafcab52fc.jpeg)

That was with Vuetify too, and we do have some stuff in our code reminiscent from the initial layout :sweat_smile: so might be that this was necessary before, and eventually after we modified the toolbar, the links didn't get an underline anymore :+1:  (at least that's my guess)